doc(server): Added documentation for LASTSAVE [format] command. (#276)
apache/kvrocks-website: docs/supported-commands.md

| Command     | Supported OR Not | Since Version | Description                                                                                                  |
| ----------- | ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------ |
| BGSAVE      | ✓                | v1.0.0        | Initiates a background save of the dataset to disk.                                                          |

> | LASTSAVE | ✓ | v2.6.0 | Returns the time of the last successful save to disk. By default ``LASTSAVE`` returns a Unix timestamp. If `ISO8601` is specified as an argument, it returns the time in ISO8601 format. |

Author: @torwig pls check now if its correct or not.

| Command     | Supported OR Not | Since Version | Description                                                                                                  |
| ----------- | ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------ |
| BGSAVE      | ✓                | v1.0.0        | Initiates a background save of the dataset to disk.                                                          |

> | LASTSAVE | ✓ | v2.6.0 | Returns the time of the last successful save to disk. By default ``LASTSAVE`` returns a Unix timestamp. If `ISO8601` is specified as an argument, it returns the time in ISO8601 format. |

Reviewer: ```suggestion
| LASTSAVE | ✓ | v2.6.0 | Returns the time of the last successful save to disk. (Additionally, `LASTSAVE ISO8601` returns the time in ISO8601 format). |
```

| ----------- | ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------ |
| BGSAVE      | ✓                | v1.0.0        | Initiates a background save of the dataset to disk.                                                          |
| LASTSAVE    | ✓                | v2.6.0        | Returns the Unix timestamp of the last successful save to disk.                                              |

> | LASTSAVE ISO8601 | ✓           | 2.11.1        | Returns the time of the last successful save to disk in ISO8601 format.

Reviewer 2: ```suggestion
| LASTSAVE ISO8601 | ✓           | unstable        | Returns the time of the last successful save to disk in ISO8601 format.
```

> | LASTSAVE ISO8601 | ✓           | 2.11.1        | Returns the time of the last successful save to disk in ISO8601 format.

Reviewer: You can add more description to the `LASTSAVE` command. This is not a new command so it should not be in a new line.

Author: | Command     | Supported OR Not | Since Version | Description                                                                                                  |
| ----------- | ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------ |
| BGSAVE      | ✓                | v1.0.0        | Initiates a background save of the dataset to disk.                                                          |
| LASTSAVE | ✓ | v2.6.0 | Returns the time of the last successful save to disk. By default, ``LASTSAVE`` returns a Unix timestamp. If `ISO8601` is specified as an argument, it returns the time in ISO8601 format. |

Something like this works? i can shorten the description if needed.

Reviewer 2: Looks, good.
